### PR TITLE
Feat/sms campaign statistics

### PR DESCRIPTION
**Pull request by JeremyDec**
| Question         | Answer
| ---------------- | ---
| Branch?          | `feat/sms-campaign-details`
| Bug fix?         | no
| New feature?     | yes
| Breaking change? | no
| Tickets          | MANAGER-5201
| License          | BSD 3-Clause

## Description

Add batch statistics page

### CODE REVIEW

File: packages/manager/modules/sms/src/sms/batches/statistics/telecom-sms-batches-statistics.controller.js [hunk 1]
```diff
@@ -0,0 +1,59 @@
...
+  $onInit() {
+    this.selectedBatch = this.batch || this.batches[0];
+
+    return this.onBatchSelect(this.selectedBatch);
+  }
+
+  onBatchSelect(batch) {
+    this.isLoadingStats = true;
+    this.batch = batch;
+
```
Comment (radireddy) [inline packages/manager/modules/sms/src/sms/batches/statistics/telecom-sms-batches-statistics.controller.js:13]: no need to set selecteBatch here?

File: packages/manager/modules/sms/src/sms/batches/statistics/telecom-sms-batches-statistics.controller.js [hunk 2]
```diff
@@ -0,0 +1,59 @@
...
+      .catch((error) => {
+        this.error = get(error, 'data.message', error.message);
+      })
+      .finally(() => {
+        this.isLoadingStats = false;
+      });
+  }
+
+  static isInError(status, statusList) {
+    return status === statusList.ERROR;
```
Comment (radireddy) [inline packages/manager/modules/sms/src/sms/batches/statistics/telecom-sms-batches-statistics.controller.js:37]: 👍

File: packages/manager/modules/sms/src/sms/batches/statistics/telecom-sms-batches-statistics.html [hunk 3]
```diff
@@ -0,0 +1,139 @@
...
+
+    <div class="row">
+        <oui-tile class="col-md-8">
+            <div class="row mb-5">
+                <div class="col-md-3">
+                    <dl class="oui-description">
+                        <dt data-translate="sms_batches_name"></dt>
+                        <dd>
+                            <oui-select
+                                match="name"
```
Comment (radireddy) [inline packages/manager/modules/sms/src/sms/batches/statistics/telecom-sms-batches-statistics.html:28]: some missing data-

**Review by radireddy — APPROVED** [review]

File: packages/manager/modules/sms/src/sms/batches/pending/cancel/routing.js [hunk 4]
```diff
@@ -0,0 +1,18 @@
+import component from '../../cancel/telecom-sms-batches-cancel.component';
+
+export default /* @ngInject */ ($stateProvider) => {
+  $stateProvider.state('sms.service.batches.pending.cancel', {
+    params: {
```
Comment (cbourgois) [inline packages/manager/modules/sms/src/sms/batches/pending/cancel/routing.js:5]: No `url` on this modal state ?

File: packages/manager/modules/sms/src/sms/batches/statistics/telecom-sms-batches-statistics.html (hunk 3 above)
Comment (Jisay) [reply packages/manager/modules/sms/src/sms/batches/statistics/telecom-sms-batches-statistics.html:28]: This is strange to see some attributes with `data-` and some without

**Review by Jisay — APPROVED** [review]

**Pull request merged**